tests/contrib/grpc/test_grpc.py

```diff
@@ -30,6 +33,25 @@ def tearDown(self):
         unpatch()
         super(GrpcTestCase, self).tearDown()
 
+    def get_spans(self, size=0, retry=20):
```

Comment:
I'd use a different name, as it might be a little bit confusing when reading code that this `assert` too.